.vib/ruby/goss/vars.yaml

```diff
@@ -0,0 +1,24 @@
+binaries:
+  - bundle
+  - bundler
+  - gem
+  - rake
+  - ruby
+modules:
+  - bundler
+  - openssl
+  - readline
+directories:
+  - mode: "0755"
+    paths:
+      - /app
```

Comment:
There are no references to this route (nor its mode) in the files. Hence, I don't think is useful and would remove this test.